### PR TITLE
Add debug request and the associated routes 

### PR DESCRIPTION
**Pull request by louneskmt**
This PR adds support for debug request (via the dashboard) and three routes to request the execution of the debug script and get the results:
```
POST   /system/debug
GET    /system/debug-result
GET    /system/debug-link
```

Related PRs:
- https://github.com/getumbrel/umbrel-dashboard/pull/275 (main)
- https://github.com/getumbrel/umbrel/pull/391

### CODE REVIEW

File: logic/system.js [hunk 1]
```diff
@@ -242,6 +242,24 @@ async function getLndConnectUrls() {
 
 }
 
+async function requestDebug() {
+    try {
+        await diskLogic.writeSignalFile(`debug`);
```
Comment (AaronDewes) [inline logic/system.js:247]:
```suggestion
        await diskLogic.writeSignalFile('debug');
```

Using ` is not needed here.

**Review by AaronDewes — DISMISSED** [review]
LGTM, only a minor suggestion

**Review by AaronDewes — DISMISSED** [review]

File: utils/const.js [hunk 2]
```diff
@@ -33,6 +33,8 @@ module.exports = {
   UPDATE_SIGNAL_FILE: process.env.UPDATE_SIGNAL_FILE || '/signals/update',
   UPDATE_LOCK_FILE: process.env.UPDATE_LOCK_FILE || '/statuses/update-in-progress',
   BACKUP_STATUS_FILE: process.env.BACKUP_STATUS_FILE || '/statuses/backup-status.json',
+  DEBUG_SIGNAL_FILE: process.env.DEBUG_SIGNAL_FILE || "/signals/debug",
```
Comment (AaronDewes) [inline utils/const.js:36]:
```suggestion
```

This is unused

**Comment by lukechilds** [comment]
Nice work!

**Pull request merged**